source/howtos/updating_pelican_sites.rst

```diff
@@ -0,0 +1,601 @@
+#.	Add/edit content
+#.	Push your branch up to Github
+#.	Make a pull request to the staging branch on Github
+#.	Solicit reviews of your changes
+#.  Merge your branch with the staging branch
+#.	Review changes on the staging site
+#.	Merge staging with the production branch
+#.  Review changes on the production site
+
+Checking out the repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The content for Pelican sites will be in a repository on Github named for the 
+site it contains:
+
+* OSL website: `osuosl-pelican`_
+* CASS website: `cass-pelican`_
+* Beaver Barcamp: `beaver-barcamp-pelican`_
+
+You can view and even edit the contents of the repository directly on its Github 
+page, but it is usually better to 'clone' the repository to your local machine.
+
+**GitHub Desktop**
+
+Open Github Desktop and log in. You will see a + button a the top left, click
+that to open the Add menu. Then click on the Clone tab. You should see a list
+of Github Organizations on the left, and a list of repositories on the right
+(the repository list may be empty if you don't have repositories of your own).
+
+.. image:: /_static/images/github_desktop-add.png
+
+Select the *osuosl* organization on the left, and find the repository you want 
+to update on the right. Type 'pelican' into the filter box to see all of our 
+Pelican site repositories.
+
+Click on your repository and then click the checkmark button that appears below. 
+Github desktop will ask you for a directory to clone to - the default is fine, 
+but feel free to put it wherever you like.
+
+Your cloned repository now appears in the left sidebar of the Github 
+application. Once the repository is added, you won't have to add it again, 
+you'll only  need this step if you want to work on a repository you haven't 
+already cloned.
+
+.. NOTE::
+  Before you start working on new changes to the site, change to 
+  the **staging** branch; this is the branch from which you should always 
+  start.
+
+At the top, you will see a pull-down containing the list of branches:
+
+.. image:: /_static/images/github_desktop-branch_select.png
+
+Select *staging*.
+
+Before you edit any of the site content, create a new branch to contain your
+own changes. Click the icon just to the left of the branch selection pull-down
+and give your branch a name, like ``your_name/a_descriptive_branch_name``.
+
+The string ``your_name/a_descriptive_branch_name`` is the full name of the
+branch, you can call it whatever you like, but it is easier to manage if you
+prefix them with your own username, followed by a descriptive name that
+explains what you are doing.
+
+For example, if I were adding Fred to the staff list, I would probably make a
+branch named ``kennric/add_fred_to_staff_list``.
+
+Double check that the 'From branch' says 'staging', and click 'Create new 
+branch'.
+
+Now you can begin making changes or adding content. Continue to the next step, 
+Editing Content.
+
+**Command Line**
+
```

Comment:
A short preamble like '*Step zero is to clone the repository into your personal projects directory. Navigate to where you usually clone repositories and run the following command:* '